shared/js/project.js

```diff
@@ -43,6 +59,7 @@ var PathPart = {
  * @property {string} level Path where this particular app type is hosted
  */
 var current;
+var currentSourceVersionId;
```

Comment:
The problem with putting it there is that it will get serialized and saved in the channel data.  We definitely don't want this token hanging around across page loads.